boa_tester/src/main.rs

```diff
@@ -348,6 +364,10 @@ struct SuiteResult {
 struct TestResult {
     #[serde(rename = "n")]
     name: Box<str>,
+    #[serde(rename = "5", default)]
+    es5: bool,
+    #[serde(rename = "6", default)]
+    es6: bool,
```

Comment:
I don't think we need an `es5/6` tag per se, we can manage with a field like `spec_version: u8`, which should allow filtering tests by `version <= 6` without having to do things like `es5 || es6`

Reply:
sounds good, I'll fix that soon